wiki/Rules/Visual_Content_Considerations/en.md

```diff
@@ -1,117 +1,52 @@
-
-In the event of a specific sort of drug abuse terminology or iconography is referenced without that drug being explicitly illegal or scheduled in most countries (such as syrup/lean/promethazine abuse), it is also considered "illicit" for the sake of this rule.
-
-Depictions of alcohol and tobacco use are allowed sparingly where such things are not portrayed in unreasonable or destructive excess.
-
-Put simply, most/all references to drug use are generally not okay and should be avoided.
-
-### Deliberately inflammatory content
-
-This includes depictions of desecration involving cultural and religious icons that are intended to deliberately inflame, such as the offensive use of Nazi or fascist symbolism (swastikas), anti-Semitic terminology and symbols, or any other icons, motifs or statements that actively encourage the harm and discrimination of another social, racial, cultural or political group.
-
-## Appropriate visual content
-
-Ideal content for use in osu! is benign and able to be consumed by an all-ages audience. When in doubt, consult the local classification for such material (if available) - if it is considered as suitable for an audience of the age of 12+ (PEGI 12 or a "PG" rating in the US and Commonwealth countries), it should largely be acceptable.
-
-Visual content **may contain some** of the following aspects in limited quantities, subject to common sense and good faith:
-
-### Swimsuits
-
-Depictions of *reasonable* swimsuits is allowed so long as said depiction is not highly excessive or in violation of the posturing and innuendo rules listed in the above section.
-
-Highly excessive is a broad term, but is generally taken to mean when an image treads well into the line of questionable content, such as when a character may be wearing a swimsuit far too small to adequately cover their physical assets, or if said physical assets command a dominating amount of the scene.
-
-Swimsuits that are too small or geared to be overtly revealing are generally not allowed if they are in good faith, considered questionable by a reasonable quantity of users.
-
-### Mild to moderate violence
-
-Scenes involving active combat, fighting or any other physical activity that may involve the presence of minor wounds or injuries is considered acceptable so long as there is not an excessive presence of gore, bloodshed, or distressing physical impairment.
+## Important things to consider
 
-### Artistic nudity
+At a basic level, we try to stick to the general idea of the [PEGI 12 classification](https://pegi.info/what-do-the-labels-mean) where possible, with special considerations for our community's needs and tastes.
 
-Any depiction of classical artwork or the rendition of the human form where the portrayal of partial nudity is done in a strictly artistic and non-sexual context may be allowed upon review from the GMT.
+All pictures should be judged as they are, without factoring in any other cases before them. Decide based on what you see.
```

Comment:
this also gives green light to media sourced from R-18 works that appears SFW by itself when cropped, is that correct?

Reply:
I think that makes sense considering nobody seems to have had issue with using sfw character art from eroge, which is in like... thousands of ranked maps

Reply:
"all media should be judged in a vacuum" applies here and any references back to NSFW content can be addressed in source or by application of a forthcoming NSFW filter